platform-common/src/main/java/com/softicar/platform/common/core/properties/SofticarHome.java

```diff
@@ -0,0 +1,25 @@
+package com.softicar.platform.common.core.properties;
+
+import java.nio.file.Path;
+
+// TODO move to somewhere else?
+// TODO javadoc
+// TODO unit test
+public class SofticarHome {
+
+	private static final String HOME_DIRECTORY_NAME = ".softicar";
+
+	// TODO javadoc
+	// TODO unit test
+	public static Path getPath() {
+
+		return Path.of(SystemPropertyEnum.USER_HOME.get(), HOME_DIRECTORY_NAME);
+	}
+
+	// TODO javadoc
+	// TODO unit test
+	public static Path getLogPath() {
+
+		return getPath().resolve("log");
```

Comment:
hard-coding of yet another path, i don't like ;)

Reply:
It's a sub directory of a path we already assume to be existent. I'd rather call this a convention.

Reply:
I we want to change this, we cannot, separating config folders from log folders is not an uncommon thing ;)